crypto/s2n_ecc.c

```diff
@@ -212,6 +215,14 @@ int s2n_ecc_compute_shared_secret_as_client(struct s2n_ecc_params *server_ecc_pa
     return 0;
 }
 
+/* Writes a shared key using a public key with a private key pair */
+int s2n_ecc_compute_shared_secret_from_params(struct s2n_ecc_params *private_ecc_params, struct s2n_ecc_params *public_ecc_params, struct s2n_blob *shared_key)
+{
+    const EC_POINT *public_ec_point = EC_KEY_get0_public_key(public_ecc_params->ec_key);
```

Comment:
let's catch the return value. The openssl impl is trivial(it just returns ec_key->pub_key) but this is one case that's worth checking for null in case it changes in the future.

Reply:
good point, will guard this.